scripts/summit/misc/runMisc.py

```diff
@@ -1,5 +1,8 @@
-import os
-dirName = os.path.dirname(__file__)
-fileToRun = os.path.join(dirName, '..', 'misc', 'runStarTrackerNightReport.py')
-print(f'Running {fileToRun}...')
-os.system(f'python {fileToRun}')
+# import os
```

Comment:
Can you comment here on this and `runMisc2.py`?  I simply do not understand why these are here.

Reply:
Of course! So, every time I add a new running service a new pod has to be created to run it in, each has an entry point that is one of these scripts. This requires manual intervention in the kubernetes recipe and redeployment by Tiago, who's not always available immediately. Having a couple of miscellaneous pods sitting around, paused, which I can redirect to run any new service I might cook up, is incredibly useful. So the workflow is that I can use these to run one of the other scripts which hasn't been added to the k8s recipe yet, and then when he gets around to it, adds a new entry point, and these go back to being spare.

Reply:
I don't know if that was terribly clear, but if not I'm happy to explain with words on Zoom or something.

Reply:
And while we don't usually leave stuff commented out, given that it's doing nothing, and makes redirection much easier for next time they come into use, I tend to just leave the cruft there, ready to be updated and reactivated.

Reply:
Can you add a comment to that effect to the files?  It'll save any future reviewer from needing to wonder about them.